components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -183,10 +114,92 @@ export default class Workspaces extends React.Component<WorkspacesProps, Workspa
                     }
                 }
                 const list = Array.from(index.values());
-                list.sort((a,b) => b.lastUse.localeCompare(a.lastUse));
+                list.sort((a, b) => b.lastUse.localeCompare(a.lastUse));
                 return list;
             }
         }
         return [];
     }
+
+    return <>
+        <Header title="Workspaces" subtitle="Manage recent and stopped workspaces." />
```

Comment:
I wonder if we should repeat the "location" in the subtitle somehow. Perhaps this was already discussed, we now have 4 /workspaces pages and takes time learn the difference between `/t/acme/ExpressDemo/workspaces` and `/t/acme/workspaces`.

Reply:
@AlexTugarev you mean something like the following?

For Personal Account:
> Manage recent and stopped workspaces.

For Teams:
> Manage recent and stopped workspaces for XYZ team.

For Projects:
> Manage recent and stopped workspaces for XYZ project.

Reply:
Is there a decision regarding this? I think the breadcrumbs should provide the needed context

Reply:
> Is there a decision regarding this? I think the breadcrumbs should provide the needed context

Let's keep this as is for now and see if breadcrumbs are sufficient in action. ✔️